samples/sample_hevc_fei/src/hevc_fei_encode.cpp

```diff
@@ -63,6 +67,7 @@ FEI_Encode::~FEI_Encode()
         {
             m_buf_allocator.Free(pQP);
         }
+        HEVCExtBRC::Destroy(m_ExtBRC);
```

Comment:
Why create called in pipeline but destroy in encode?

Reply:
Create is called in pipeline because it depends on input params, when Destroy doesn't. 
Also, destroyable pointer is stored in encode, not in pipeline.

Do you think it's better to move destroy step to pipeline destructor?

Reply:
I think that Create should be called in Encode because it's Encode's attribute, not pipeline